_events/2020-03-19-wirvsvirus.md

```diff
@@ -0,0 +1,13 @@
+---
+site: freiburg
+tags: [hackaton]
+title: "WirVsVirus Hackathon"
+starts: 2020-03-20
+ends: 2020-03-22
+organiser:
+  name: Bundesregierung
+  email: info@wirvsvirushackathon.org
+location: online
+---
+
+The Federal Government of Germany is organising the [#WirVsVirus Hackathon](https://wirvsvirushackathon.org/).
```

Comment:
a little bit more encouraging ... please feel free to join ... etc... We offer jupyter notebook, RStudio instances, link to the training mterial about it ...

Link to this project and more importantly to the data ...
https://covid19.galaxyproject.org/

Reply:
please add add the support email adress if people need resources they should contact us ...

Reply:
We have only 32 slots free for IT. Should I start to free other cloud nodes?